example/spec/syncGranule/SyncGranule.output.payload.template.json

```diff
@@ -10,15 +10,15 @@
           "filename": "",
           "fileSize": 17865615,
           "path": "cumulus-test-data/pdrs",
-          "url_path": "{cmrMetadata.Granule.GranuleUR}/{file.name}"
+          "url_path": "{cmrMetadata.Granule.Collection.ShortName}/{extractYear(cmrMetadata.Granule.Temporal.RangeDateTime.BeginningDateTime)}/{substring(file.name, 0, 3)}"
```

Comment:
@jennyhliu @laurenfrederick 
Was this change actually tested and passing in an integration test? 
I know the new url_path is already set in collection. But that collection.url_path doesn't get set in the specific file until post-to-cmr (and now move-granules) but definitely Not in sync granules. So the output file.url_path would still be "{cmrMetadata.Granule.GranuleUR}/{file.name}"

Reply:
@kkelly51 The integration-test does work.  I just created a PR from granule-storage for testing purpose.
https://circleci.com/gh/cumulus-nasa/cumulus/2237?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link